namespace/namespace_host.go

```diff
@@ -15,6 +15,7 @@ func ApplyNamespaceHost(
 	deckfile *yaml.Node, // the deckFile to operate on
 	selectors yamlbasics.SelectorSet, // the selectors to use to select the routes
 	hosts []string, // the hosts to add to the routes
+	//nolint:predeclared
```

Comment:
It could be because of the differences in the golangci-lint version we are using in the CI and the one used locally.
Let me check with same versions, because I added all the //nolint comments after checking the CI. I wasn't getting any warnings in my local setup.